runtime/src/main/java/org/corfudb/runtime/object/DiskBackedSMRSnapshot.java

```diff
@@ -24,6 +28,15 @@
 @Slf4j
 public class DiskBackedSMRSnapshot<S extends SnapshotGenerator<S>> implements SMRSnapshot<S> {
 
+    private static final String SNAPSHOT_CREATED_METRIC = "snapshots.created";
+    private static final String SNAPSHOT_RELEASED_METRIC = "snapshots.released";
+    private static final String OBJECT_ID_TAG = "objectId";
+
+    // Need to keep track of accumulative values. MicroMeter's Counter does
+    // not suffice here as it only keeps track of the rate
+    private static final ConcurrentHashMap<UUID, AtomicLong> createdSnapshots = new ConcurrentHashMap<>();
```

Comment:
hi @vjeko - how are these counters going to get reset? If not reset won't these values build up over time and cause heap dumps?